<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButModificar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>580</y>
     <width>441</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string> Mi filtro</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QDial" name="dialR">
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDial" name="dialG">
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDial" name="dialB">
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>            Rojos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>           Verdes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>            Azules</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="radioButFiltrar">
          <property name="text">
           <string>Filtrar</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButNoFiltrar">
          <property name="text">
           <string>No filtrar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButSobrePoner">
          <property name="text">
           <string>Sobreponer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Agregar imagen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButAbrirImagen">
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>781</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="imagen">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="imagenModificada">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>451</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Efectos</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="Filtros">
        <item>
         <widget class="QRadioButton" name="radioButAnd">
          <property name="text">
           <string>HDR</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButWalden">
          <property name="text">
           <string>Walden</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButLux">
          <property name="text">
           <string>Lux</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButAmaro">
          <property name="text">
           <string>Amaro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButHudson">
          <property name="text">
           <string>Hudson</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButOr">
          <property name="text">
           <string>Blanco</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButRojos">
          <property name="text">
           <string>Rojo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButRojointenso">
          <property name="text">
           <string>Rojo intenso</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButVerdes">
          <property name="text">
           <string>Verde</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButVerdeIntenso">
          <property name="text">
           <string>Verde intenso</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QRadioButton" name="radioButXor">
          <property name="text">
           <string>Azul</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButAzulintenso">
          <property name="text">
           <string>Azul intenso</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButRyV">
          <property name="text">
           <string>Rojo con verde</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButGris">
          <property name="text">
           <string>Grises</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButMiFiltro">
          <property name="text">
           <string>Mi filtro</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
